types/node/index.d.ts

```diff
@@ -38,6 +38,7 @@
 //                 Thanik Bhongbhibhat <https://github.com/bhongy>
 //                 Marcin Kopacz <https://github.com/chyzwar>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
```

Comment:
This will for sure cause a lot fails with other modules as definition depending on node (>1500) would need the same min requirement.